### PR TITLE
Chore/resolve conflict

### PR DESCRIPTION
**Pull request by aatbip**
### Task/Ticket

[Conflict resolved]

#### The main changes are

- [x] Resolved unnecessary conflicts with main

### CODE REVIEW

File: src/app/api/client/route.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  const options = {
+    method: 'GET',
+    headers: {
+      accept: 'application/json',
+      'X-API-KEY': process.env.COPILOT_API_KEY as string,
+    },
+  }
+
+  const res = await fetch(
+    `https://api-beta.copilot.com/v1/clients/${clientId}`,
```
Comment (saugatmaharjan) [inline src/app/api/client/route.ts:15]: why is this api url static? is this same for all envs? would it be safe to have an env variable ?

File: src/app/client-preview/page.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+  const company = await getCompany(_client.companyId)
+
+  const template = Handlebars?.compile(settings.content)
+  const client = {
+    ..._client,
+    company: company.name,
+  }
+
+  const htmlContent = template({ client: client })
```
Comment (saugatmaharjan) [inline src/app/client-preview/page.tsx:44]: `{ client }`

File: src/components/tiptap/bubbleMenu/DropdownBubbleMenu.tsx [hunk 3]
```diff
@@ -31,19 +31,19 @@ const DropdownBubbleMenu: FC<IBubbleMenuContainer> = ({ editor }) => {
     const name = parent.type.name
 
     if (name === 'heading' && level === 1) {
-      setSelectedFormatter(Formatter.h1)
+      setSelectedFormatter('Heading 1')
```
Comment (saugatmaharjan) [inline src/components/tiptap/bubbleMenu/DropdownBubbleMenu.tsx:34]: why revert all these?

**Comment by aatbip** [comment]
@saugatmaharjan Commits 58eca32fd26eafce037846dc3345cebc670f86cf, a1cf8cad8e42b9f638424d60fbefd639bcdcbc32, 984507ec5d8263a523342366f0072e00ec8e248d address the reviews.

File: src/app/api/autofill/route.ts [hunk 4]
```diff
@@ -10,7 +10,7 @@ export async function GET() {
   }
 
   const res = await fetch(
-    'https://api-beta.copilot.com/v1/custom-fields',
+    `${process.env.COPILOT_API_URL}/v1/custom-fields`,
```
Comment (sazanrjb) [inline src/app/api/autofill/route.ts:13]: lets create this in config file, we seem to have taken this in other places as well. 

config/app.ts
`export const  copilotAPIUrl: process.env.COPILOT_API_URL`

File: src/app/api/client/route.ts [hunk 5]
```diff
@@ -0,0 +1,20 @@
+import { NextResponse, NextRequest } from 'next/server'
+
+export async function GET(req: NextRequest) {
+  const clientId = req.nextUrl.searchParams.get('clientId') as string
+
+  const options = {
+    method: 'GET',
+    headers: {
+      accept: 'application/json',
+      'X-API-KEY': process.env.COPILOT_API_KEY as string,
```
Comment (sazanrjb) [inline src/app/api/client/route.ts:10]: replace from config

**Review by sazanrjb — APPROVED** [review]

**Pull request merged**